src/main/resources/templates/crossref/body.ftlx

```diff
@@ -56,7 +56,7 @@
 										<suffix>${pubContributor.contributor.suffix}</suffix>
 									</#if>
 									<#if pubContributor.contributor.orcid??>
-										<ORCID>${pubContributor.contributor.orcid}</ORCID>
+										<ORCID>https://orcid.org/${pubContributor.contributor.orcid}</ORCID>
```

Comment:
Can we use DataNormalizationUtils.ORCID_PREFIX or ${pubContributor.contributor.denormalizeOrcid} instead of hardcoding the url in a second place?